build-wheelhouse/action.yml

```diff
@@ -192,14 +193,35 @@ runs:
         fi
         echo "wheelhouse_target=$( [[ "${TARGET}" == '' ]] && echo 'wheelhouse' || echo "${TARGET}-wheelhouse")" >> ${GITHUB_OUTPUT}
 
+    - name: "Install poetry if needed (and configure it as needed - no auto poetry env + pipx + dedicated virtualenv)"
+      if: steps.build-system.outputs.BUILD_BACKEND == 'poetry'
+      id: poetry-env
+      shell: bash
+      env:
+        PIPX_BIN_DIR: ${{ runner.temp }}/pipx/bin
+        PIPX_HOME: ${{ runner.temp }}/pipx/home
+      run: |
+        python -m pip install pipx
+        python -m pipx install poetry
+        echo "${{ runner.temp }}/pipx/bin" >> $GITHUB_PATH # zizmor: ignore[github-env]
+        echo "POETRY_VIRTUALENVS_CREATE=false" >> $GITHUB_ENV
+        python -m venv .venv
+        if [[ ${{ runner.os }} == 'Windows' ]]; then
+          echo "activate_env=$(echo 'source .venv/scripts/activate')" >> ${GITHUB_OUTPUT}
```

Comment:
Should this use the script for powershell? The one named `Activate.ps1`? I am not that familiar with windows runners.

Reply:
If we make use of the ``shell: bash`` it is not needed =)